CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+7. Make changes in your local project, including adding files or updating existing code.
+8. Add the changed files and create a commit message describing the changes.
+e.g. `git add README.md` or `git add -A` for all changed files
+e.g. `git commit -m "update table of contents in readme"`
+_Note: more information on [writing a good commit message](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) can be found in the linked article_
+9. Continue with 6 and 7 above until all changes are complete.
+
+__When the changes are complete and you are ready to submit them to our project__, please follow the additional steps below.
+
+10. Check that all of your submissions follow the relevant [Style Guides](#style-guides).
+11. Push the changes to the origin repository on Github (your fork).
+`git push -u origin <branch-name>`
+_Note: the `-u` is only required when first pushing a new branch to the origin (your fork)._
+12. On github, [submit the pull request (PR)](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) to the upstream repository when changes are complete.
+13. Follow all of the instructions in the [Pull Request Template]().
+
+### Staying Up to Date
+
+__It is important to always work from the most up to date code in our repository.__ Before making any changes, always be sure that your local repository contains the most recent changes from the upstream repository (our project).
+
+To update your local repository, follow the steps below.
+
+1. Check what branch you are currently working on.
+`git status`
+2. Check out the master branch of your local repository.
+`git checkout master`
+3. Fetch changes from the upstream master repository (ours).
+`git fetch upstream master`
+4. Rebase the changes from the upstream master respository into your local repository.
+`git rebase upstream/master`
+5. Check out a new branch to make and submit changes, following the procedures in [Contributing Code](#contributing-code) starting from Step 5.
+
+## Style Guides
+
+### Naming Git Branches
+
+__Use the branch naming structure of "verb/intent"__, and examples are listed below. This helps make branch names easier to read, understand, and follow for the project maintainers.
+
+```
+upd/resource-list
+feat/omni-auth
+fix/auth-flow
+```
+
+### Creating Git Commit Messages
+
+__Keep git commit messages short, simple, and informative__, ideally under 50 characters.
+
+__Write commit messages in the imperative__, using "fix bug" and not "fixed bug" or "fixes bug". This convention matches up with commit messages generated by commands like `git merge` and `git revert`.
+
+`git commit -m "update contributing.md with new sections"`
+
+### Front End Style Guide
+
+📝 Use Material-UI to build and style components.
+
+📝 When styling, use Material-UI styling APIs.
```

Comment:
We can probably remove 1 of these, I think they're saying the same thing.

Reply:
Fixed per c7bdd41, removed first line.